Task 2

```diff
@@ -0,0 +1,24 @@
+var text = "Lorem Ipsum is simply dummy text of the printing and typesetting industry."+
+"To offer protection against timing attacks and fingerprinting, the precision of "+
+"performance now might get rounded depending on browser settings."+
+"In Firefox, the privacy.reduceTimerPrecision  preference is enabled "+
+"by default and defaults to ms.";
+
+var lstFrequency = {};
+
+function getFrequency(){
+  for(var index=0;index<text.length;index++){
```

Comment:
What if I want to compute frequency of a different text?
What if there were no alphabetic characters in the text?